sdk/storage/azure-storage-queue/azure/storage/queue/_encryption.py

```diff
@@ -866,7 +886,7 @@ def get_blob_encryptor_and_padder(cek, iv, should_pad):
     return encryptor, padder
 
 
-def encrypt_queue_message(message, key_encryption_key, version):
+def encrypt_queue_message(message: object, key_encryption_key: object, version: str) -> str:
```

Comment:
Fix "messge", use a `message_bytes`, comes in as a `str` so lets make it that way + in the docstring